docusaurus/docs/installation.md

```diff
@@ -114,8 +114,34 @@ Reassure CLI will automatically try to detect your source code branch name and c
 ```sh
 yarn reassure --branch [branch name] --commit-hash [commit hash] 
 ```
+## CI setup
+To make setting up the CI integration and all prerequisites more convenient, we have prepared a CLI command which will generate all necessary templates for you to get started with. 
 
-## Write performance testing script
+Simply run:
+```bash
+yarn reassure init --logLevel <verbose|default|silent>
+```
+This will generate the following file structure
+```
+├── <ROOT>
+│   ├── .reassure
+│   │   └──  reassure-tests.sh
```

Comment:
Why is reassure-tests.sh put under .reassure? reassure-tests.sh should be. top level.